tests/integration/tests/enhancements/with-policy/auth.test.ts

```diff
@@ -505,4 +505,33 @@ describe('With Policy: auth() test', () => {
             ])
         );
     });
+
+    it('Default auth() without user context', async () => {
+        const { enhance } = await loadSchema(
+            `
+        model User {
+            id String @id
+            posts Post[]
+
+            @@allow('all', true)
+        }
+
+        model Post {
+            id String @id @default(uuid())
+            title String
+            author User @relation(fields: [authorId], references: [id])
+            authorId String @default(auth().id)
+
+            @@allow('all', auth().id != null)
```

Comment:
This is confusing, I should have used `@@allow('all', true)`, but it doesn't change the test.